### DISCUSSION TITLE
Possible to output a registry file of net registry changes?

### DISCUSSION THREAD

**Discussion by Coldblackice** [sandboxie-plus/Sandboxie#1917] [Various ideas and suggestions]
Not sure if there's already a way to do this (I couldn't find one), but would it be possible to add the ability to export a registry file (.REG) of the net changes done to the registry by a sandboxed program?

For example, if a sandboxed program creates `HKLM\Software\YaddaNEW\` and deletes `HKLM\Software\YaddaOLD\`, Sandboxie would offer the option to export a .REG file of those changes, albeit translated to actual OS registry locations, not sandboxed/virtualized equivalents. If a user were to then import that .REG file, their system registry would be modified with the equivalent changes made by the sandboxed program.

Currently, I do this manually by either monitoring the sandboxed process with process-monitor, or just manually trawling through the HKU\Sandbox_DefaultBox key, but it would be great if there was a function that could do this automatically.

**Comment by DavidXanatos** [comment]
this would be great indeed and in principle it would only need to parse the sandboxed reg hive and adapt the key paths

**Comment by DavidXanatos** [comment]
i made an issue for that so i dont forget: https://github.com/sandboxie-plus/Sandboxie/issues/1918

**Reply by Coldblackice** [reply]
Awesome! This is a feature I would utilize all the time, looking forward to it!
